tests/src/container-smoke.spec.ts

```diff
@@ -0,0 +1,129 @@
+/**********************************************************************
+ * Copyright (C) 2023 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { Page } from 'playwright';
+import { afterAll, beforeAll, test, describe } from 'vitest';
+import { expect as playExpect } from '@playwright/test';
+import { PodmanDesktopRunner } from './runner/podman-desktop-runner';
+import { WelcomePage } from './model/pages/welcome-page';
+import { NavigationBar } from './model/workbench/navigation';
+import { delay, waitWhile } from './utility/wait';
+import { deleteContainer, deleteImage } from './utility/operations';
+import { removeFolderIfExists } from './utility/cleanup';
+import { join } from 'path';
+
+let pdRunner: PodmanDesktopRunner;
+let page: Page;
+const imageToPull = 'alpine';
+const imageTag = 'latest';
+const containerToRun = 'alpine';
+
+beforeAll(async () => {
+  console.log('BeforeAll containers');
+  await removeFolderIfExists(join('tests', 'output', 'podman-desktop'));
+  pdRunner = new PodmanDesktopRunner();
+  page = await pdRunner.start();
+  const welcomePage = new WelcomePage(page);
+  await welcomePage.handleWelcomePage(true);
+  // hard wait giving a time to podman desktop to load up
+  await delay(5000);
```

Comment:
IMHO the delay to get UI ready is broken, we should wait that an element is displayed but not only 'wait'

Reply:
I would generally agree, although in this case, I am not sure for what exactly to wait for. UI is up, it only seems that I need to wait for a store to reload so that I can work with images. It takes around 3-4 sec. on local machine after app is all ready to notice images. But in any case I cannot exclude there exists better waiter. will look at it.

Reply:
there is a troubleshooting page with all stores listed
you could wait here the value you expect

Reply:
Issue for this purpose. https://github.com/containers/podman-desktop/issues/3417